source/codegen/metadata/nirfmxspecan_restricted/config.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+config = {
+    "api_version": "21.0.0",
+    "restricted_driver_service": True,
```

Comment:
NOTE: New key to support functions exported from driver DLL but not in public header.